draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -1021,9 +1134,12 @@
       is not the case, the TLV should be considered malformed, and the
       "Treat-as-withdraw" procedure of <xref target="RFC7606"/> is
       applied.</t>
+      <!-- XXX JMH - this potentially addresses my question about the
+      "reserved" field at the front of the new attribute.  Is that your intent
+      here? -->
```

Comment:
yes.

Reply:
That's what I was afraid of.
It's not practice in BGP path attributes to go significantly out of your way to try to align initial PDU contents.  What you've effectively defined in this document is a path attribute that is:
attr-type/attr-flags/attr-length:
RESERVED
Sequence of regular TLVs.

The "RESERVED" is weird.  What would you do that can't be encoded in the future in one of the TLVs?

Reply:
now with the Metadata Path TLV being removed, replaced with the following, is it still an issue? 

_"The Metadata Path Attribute is an optional non-transitive BGP Path
        attribute that carries metrics and metadata about the edge services
        attached to the egress router. The Metadata Path Attribute, to be
        assigned by IANA <xref target="RFC2042"/>, consists of a set of
        Sub-TLVs, and each Sub-TLV contains information for specific metrics
        of the edge services."_

Reply:
Replaced with this text.  We'll need to re-audit for the leading RESERVED text to make sure it's fully gone.  Meanwhile, resolving this item.